[2022-07-22 07:37:53,401] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: etl.get_data manual__2022-07-22T07:37:51.802156+00:00 [queued]>
[2022-07-22 07:37:53,407] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: etl.get_data manual__2022-07-22T07:37:51.802156+00:00 [queued]>
[2022-07-22 07:37:53,408] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-07-22 07:37:53,409] {taskinstance.py:1377} INFO - Starting attempt 1 of 1
[2022-07-22 07:37:53,409] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-07-22 07:37:53,416] {taskinstance.py:1397} INFO - Executing <Task(_PythonDecoratedOperator): get_data> on 2022-07-22 07:37:51.802156+00:00
[2022-07-22 07:37:53,420] {standard_task_runner.py:52} INFO - Started process 13152 to run task
[2022-07-22 07:37:53,422] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'etl', 'get_data', 'manual__2022-07-22T07:37:51.802156+00:00', '--job-id', '48', '--raw', '--subdir', 'DAGS_FOLDER/pipeline_2.py', '--cfg-path', '/tmp/tmprzeqk6bc', '--error-file', '/tmp/tmpzwpz40ig']
[2022-07-22 07:37:53,423] {standard_task_runner.py:80} INFO - Job 48: Subtask get_data
[2022-07-22 07:37:53,462] {task_command.py:371} INFO - Running <TaskInstance: etl.get_data manual__2022-07-22T07:37:51.802156+00:00 [running]> on host b51f0a1d5848
[2022-07-22 07:37:53,501] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=etl
AIRFLOW_CTX_TASK_ID=get_data
AIRFLOW_CTX_EXECUTION_DATE=2022-07-22T07:37:51.802156+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-07-22T07:37:51.802156+00:00
[2022-07-22 07:37:53,602] {base.py:68} INFO - Using connection ID 'pg' for task execution.
[2022-07-22 07:37:53,611] {python.py:173} INFO - Done. Returned value was: None
[2022-07-22 07:37:53,619] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=etl, task_id=get_data, execution_date=20220722T073751, start_date=20220722T073753, end_date=20220722T073753
[2022-07-22 07:37:53,635] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-07-22 07:37:53,657] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
